docs/io-app/view-message-list.puml

```diff
@@ -0,0 +1,54 @@
+@startuml name ViewMessageList-ioApp
+participant MessagesHomeScreen
+participant PagerViewContainer
+participant MessageList
+participant Redux
+participant allPaginatedReducer
+participant HandleReloadAllMessages
+note right of HandleReloadAllMessages
+ saga
+end note
+MessagesHomeScreen -> PagerViewContainer : contains
```

Comment:
what is the meaning of `contains` in this context ?

Reply:
Removed and specified with notes